server/sonar-server/src/main/java/org/sonar/server/rule/index/RuleIndex.java

```diff
@@ -91,104 +90,73 @@
         Arrays.asList(RuleStatus.values()),
         new Function<RuleStatus, String>() {
           @Override
-          public String apply(RuleStatus input) {
+          public String apply(@Nonnull RuleStatus input) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The "input" parameter nullability is different in the superclass method, and that should not be changed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2638)